### PR TITLE
Add what I learned today

### PR DESCRIPTION
**Pull request by JoshSJKim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
 # Learn Git
 
 ## What to do next...
+
+## Today I learned...
```
Comment (sidhlee) [inline README.md:5]: Are dots at the end necessary..?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+
+- git push/pull
+- Do Not push directly to main
```
Comment (sidhlee) [inline README.md]: maybe add a reason too..?

File: README.md (hunk 1 above)
Comment (JoshSJKim) [reply README.md:5]: Yes

**Review by sidhlee — APPROVED** [review]
LGTM 🚀

**Pull request merged**